docs/components/concepts/document-handling.md

```diff
@@ -0,0 +1,43 @@
+---
+id: document-handling
+title: "Document handling"
+description: "Learn more about integration, secure management, and efficient storage and retrieval of documents across development and production environments."
+keywords: ["document handling"]
+---
+
+Offering a more comprehensive approach to document handling, Camunda now allows for easier integration, secure management, and efficient storage and retrieval of documents across both development and production environments.
+
+Whether you are working across Connectors, Modeler, Forms, Operate, or Tasklist, document handling may be beneficial for users who want the following:
+
+- [AWS S3](https://aws.amazon.com/s3/) storage and bucket creation per cluster to securely store and retrieve documents in an external, scalable storage solution, and to ensure storage is properly isolated and managed for each environment.
+- Set document limitations and access permissions, ensuring compliance and secure handling of sensitive documents within workflows.
+- Support for the local file system as a storage option for development and testing purposes to set up and manage documents in local environments.
+- Preview, track, and manage documents associated with process instances.
+
+Below, learn about capabilities of document handling for each related component:
+
```

Comment:
NOTE: Everything below this point is not yet documented, and merely reflects an outline for the expected changes.